fastapi_crudrouter/core/sqlalchemy.py

```diff
@@ -95,7 +132,28 @@ def route(
             else:
                 raise NOT_FOUND from None
 
-        return route
+        async def async_route(
+            item_id: self._pk_type, db: Session = Depends(self.db_func)  # type: ignore
+        ) -> Model:
+            model: Model
+            try:
+                (model,) = (
+                    await db.execute(
+                        select(self.db_model).where(self.db_model.id == item_id)
```

Comment:
Just fyi... my team noticed this doesn't work with PK's not named "id". I think this code might work better:

```suggestion
                        select(self.db_model).where(getattr(self.db_model, self._pk) == item_id)
```